SPEC.md

```diff
@@ -552,19 +552,35 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ## Types
 
-### AC Name Type
+### AC Identifier Type
 
-An AC Name Type is restricted to lowercase characters accepted by the DNS [RFC1123](http://tools.ietf.org/html/rfc1123#page-13) and "/".
-An AC Name Type cannot be an empty string and must begin and end with an alphanumeric character.
-An AC Name Type will match the following [RE2](https://code.google.com/p/re2/wiki/Syntax) regular expression: `^[a-z0-9]+([-./][a-z0-9]+)*$`
+An AC Identifier Type is restricted to lowercase URI unreserved characters defined in [RFC3986](http://tools.ietf.org/html/rfc3986#section-2.3)
+An AC Identifier Type cannot be an empty string and must begin and end with an alphanumeric character.
+An AC Identifier Type will match the following [RE2](https://code.google.com/p/re2/wiki/Syntax) regular expression: `^[a-z0-9]+([-._~/][a-z0-9]+)*$`
```

Comment:
so `v~-_._-~v` is a valid identifier.  Interesting.  We specifically said that identifiers are a relatively loose string (like you spec) preceeded by an optional DNS name.  Specifically, the existence of a / at all implies that the first part is a DNS subdomain.  It's a little tighter than this - maybe this is OK.  Just go through and think about the most ridiculous valid string a user could input.